server/monitor/src/main/java/org/apache/accumulo/monitor/rest/tservers/TabletServers.java

```diff
@@ -23,11 +23,15 @@
 
 import org.apache.accumulo.monitor.rest.manager.ManagerInformation;
 
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
 /**
  * Generates a list of servers, bad servers, and dead servers
  *
  * @since 2.0.0
  */
+@SuppressFBWarnings(value = "PA_PUBLIC_PRIMITIVE_ATTRIBUTE",
+    justification = "class used for serialization")
```

Comment:
Why can't the fields here be private? Seems they are only used in this class anyways